### PR TITLE
fix: Restrict seed file to handle test and dev environments

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

Restrict seed file to handle test and dev environments

### CODE REVIEW

File: priv/repo/seeds.exs [hunk 1]
```diff
@@ -1,7 +1,10 @@
 alias Realtime.{Api.Tenant, Repo}
 import Ecto.Adapters.SQL, only: [query: 3]
 
+IO.inspect(Mix.env())
```
Comment (chetan) [inline priv/repo/seeds.exs:4]: debug?

**Review by chetan — CHANGES_REQUESTED** [review]

File: priv/repo/seeds.exs [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+
 tenant_name = "realtime-dev-tenant"
```
[the remark this answers is not in the document]
Comment (filipecabaco) [reply priv/repo/seeds.exs:6]: you are right, I'll doublecheck why this value was set

File: priv/repo/seeds.exs (hunk 1 above)
Comment (filipecabaco) [reply priv/repo/seeds.exs:4]: 🤦 yeah removing

File: priv/repo/seeds.exs (hunk 2 above)
[the remark this answers is not in the document]
Comment (filipecabaco) [reply priv/repo/seeds.exs]: checked on other repos and indeed does not make sense to have this value from what I gather, changed it to the value `realtime-dev`

**Review by chetan — APPROVED** [review]
LGTM

**Review by w3b6x9 — APPROVED** [review]
lgtm!

**Pull request merged**